frontend/src/containers/HomePage.js

```diff
@@ -79,7 +52,7 @@ export class HomePage extends Component {
           <div className='heading'>Collect money for your community, transparently.</div>
         </section>
         <section className='HomePageInfo' id="howitworks">
-          <div className='heading'>What is a Open Collective?</div>
+          <div className='heading'>What is an Open Collective?</div>
```

Comment:
I'd argue we should write "an open collective" (or "Open Collective" should refer to the company)